docs/source/hardware/build_materials.rst

```diff
@@ -67,4 +67,18 @@ DC Power Plug         `Amazon <https://www.amazon.com/SIM-NAT-Pigtails-Security-
 Assembly Guide
 ----------------------
 
-TBD
\ No newline at end of file
+#. Attach a MT2213 935KV motor to each F330 arm.
+#. Connect a ZTW Spider ESC to each motor. The ESCs should be physically attached to the arms with plastic tape so they do not interfere with propellers.
+#. Solder two 10 Gauge wires (red and black) to the bottom plate of F330, connecting the end of both wires to a XT 60 connector for battery connection.
+#. Solder two 16 Gauge wires (red and black) to the bottom plate of F330, on top of where the 10 Gauge wires are soldered. The other end of the wires are soldered to　the input+/- terminals of the power distribution board for PixFalcon.
```

Comment:
there's a non-space character at "to the" in Step 4, in the sentence "The other end of the wires are soldered to the input+/-..."

Reply:
Fixed.